nxc/protocols/smb.py

```diff
@@ -1565,7 +1636,7 @@ def dpapi(self):
                 credential.url,
             )
 
-        if not (credentials and system_credentials and browser_credentials and cookies and vaults and firefox_credentials):
+        if not (credentials or system_credentials or browser_credentials or cookies or vaults or firefox_credentials):
```

Comment:
Why that change? Originally i intended to just print that fail message if none credentials were found at all.

Reply:
It the previous line of code, It would have written the error message if at least one folder was empty.
NOT (A and B) = NOT A or NOT B
In this case we want the opposite, NOT A and NOT B => NOT (A or B)

Reply:
boolean logic puiffffff ^^

Reply:
Oh yes you are right